ext.js

```diff
@@ -1,33 +1,46 @@
+((function extScope() {
+  function handleMessage(e) {
+    try {
+      if (e.data && e.data.height.length > 2 && e.data.request === 'changeHeight') {
+        console.log('resizing to : ' + e.data.height);
+        var iframe = findIframe(e.data.href);
+        if (iframe) {
+          iframe.style.height = e.data.height;
+       }
+        else {
+          console.log('Could not find iframe with href', e.data.href);
+        }
+      } else {
+        console.log('Missing e.data or e.data.request or proper e.data.height');
+      }
+    } catch (e){
+      console.log('Caught error: ' , e, e.stack);
+    }
+  }
 
-
-window.addEventListener('message', handleMessage, false);
-
-function handleMessage(e) {
-  try {
-    if (e.data && e.data.height.length > 2 && e.data.request === 'changeHeight') {
-      console.log('resizing to : ' + e.data.height);
-      var iframe = findIframe(e.data.href);
-      if (iframe) {
-        iframe.style.height = e.data.height;
-     }
-      else {
-        console.log('Could not find iframe with href', e.data.href);
+  function findIframe(href) {
+    var iframes = document.querySelectorAll('iframe');
+    for (var i = 0; i < iframes.length; ++i) {
+      var iframe = iframes[i];
+      if (iframe.src === href) {
+        return iframe;
       }
-    } else {
-      console.log('Missing e.data or e.data.request or proper e.data.height');
     }
-  } catch (e){
-    console.log('Caught error: ' , e, e.stack);
   }
-}
 
-function findIframe(href) {
-  var iframes = document.querySelectorAll('iframe');
-  for (var i = 0; i < iframes.length; ++i) {
-    var iframe = iframes[i];
-    if (iframe.src === href) {
-      return iframe;
+  function hideSyllabusTableIfFlagExists() {
+    var flag = document.querySelector('#hide-syllabus-table');
```

Comment:
Heh, I don't think they have a style:

<img width="746" alt="screenshot 2015-07-15 16 09 47" src="https://cloud.githubusercontent.com/assets/324298/8708786/092c2eb0-2b0c-11e5-9ce2-0c33f1308e22.png">


But I do agree in a broad sense re: matching host style

Incidentally, though, in the case of `id` values, [browsers are not expected to distinguish](http://www.w3.org/TR/html401/struct/links.html#h-12.2.1) between `myId` and `MYID`. I bet plenty of them do, but since it's not part of the standard, I avoid camelCasing in id and class names.